### DISCUSSION TITLE
It possible to prevent sandboxie drag my mouse across the monitor in-game?

### DISCUSSION THREAD

**Discussion by nos4r3** [sandboxie-plus/Sandboxie#4272] [General]
I don't know why but the game i play is once human

When playing with just 1 client in my main monitor while open second monitor to do another stuff in fullscreen my mouse never go across the screen at all it will stay lock in the main game i focus

But when i'm using sandboxie to open steam + second once human client i move this window to second monitor now the problem start here when playing with my main client, while playing with main monitor if i move mouse to the right too much my mouse will just go across to second monitor and it really hard to play like this when i close sandboxie second steam everything will work fine again

Is there anyway to fix or prevent this?

Using DisplayFusion to lock mouse in the first monitor also not perfect solution because i still can click to second monitor until displayfusion return cursor to main monitor

**tldr my mouse are not lock in the game from the main monitor when using sandboxie i want to prevent this any idea?**

**Comment by bastik-1001** [comment]
If you use Sandboxie-Plus 1.13.6 or newer, you may try by adding `BlockInterferenceControl=y` to the sandbox configuration, even though this might not work well with games.

Edit: I guess, neither the game nor steam is causing this, so I don't assume that this will work. It could be an issue of Sandboxie.

**Comment by offhub** [comment]
You might try using `OpenWinClass=#` or `OpenWinClass=*` setting.

> https://sandboxie-plus.github.io/sandboxie-docs/Content/OpenWinClass/

**Comment by nos4r3** [comment]
I try both solutions

First one sandboxing can't launch steam at all

Second one i try OpenWinClass=* steam will stop at sign-in screen and it only appear at black window only ( it happen when i'm using sandboxie normally too when it happen i have to wait it out until sandboxie promp me to pick option when i'm pick option restart without sandboxing browser or smth it will work 100% of the time so i kinda hope to know how to make sandboxie launch without sandboxie browser thingy too

OpenWinClass=# steam and game work normally but it still can't contain mouse in main screen

**Reply by offhub** [reply]
Try specifying the program in the settings, for example: `OpenWinClass=game.exe,*`.

For any Steam-related issues, check out this link:
https://github.com/sandboxie-plus/Sandboxie/issues?q=is%3Aissue+Steam+
